ownCloud/ownCloud/Bookmarks/Connection Issues /IssuesViewController.swift

```diff
@@ -0,0 +1,217 @@
+            self.issues = issues!
+        }
+        self.setupTransitions()
+    }
+
+    override func dismiss(animated flag: Bool, completion: (() -> Void)? = nil) {
+        super.dismiss(animated: flag, completion: completion)
+
+        if let completionHandler = completionHandler {
+            completionHandler()
+        }
+    }
+
+    private func setupTransitions() {
+        self.modalPresentationStyle = .custom
+        let transitioningDLG = IssuesTransitioningDelegate()
+        self.modalPresentationVC = transitioningDLG
+        self.transitioningDelegate = transitioningDLG
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor.black.withAlphaComponent(0.5)
+        issuesTableView.delegate = self
+        issuesTableView.dataSource = self
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        bottomContainer.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(bottomContainer)
+
+        bottomContainer.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor, constant: -20).isActive = true
+        bottomContainer.leftAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leftAnchor, constant: 20).isActive = true
+        bottomContainer.rightAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.rightAnchor, constant: -20).isActive = true
+        bottomContainer.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        bottomContainer.contentHuggingPriority(for: .vertical)
+
+        issuesTableView.translatesAutoresizingMaskIntoConstraints = false
+        issuesTableView.layer.cornerRadius = 10
+        issuesTableView.backgroundColor = UIColor(hex: 0xF2F2F2)
+        issuesTableView.bounces = false
+        issuesTableView.showsVerticalScrollIndicator = false
+
+        self.view.addSubview(issuesTableView)
+
+        issuesTableView.bottomAnchor.constraint(equalTo: bottomContainer.topAnchor, constant: -20).isActive = true
+        issuesTableView.leftAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leftAnchor, constant: 20).isActive = true
+        issuesTableView.rightAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.rightAnchor, constant: -20).isActive = true
+        issuesTableView.topAnchor.constraint(greaterThanOrEqualTo: self.view.safeAreaLayoutGuide.topAnchor, constant: 20).isActive = true
+        tableHeighConstraint = issuesTableView.heightAnchor.constraint(equalToConstant: issuesTableView.contentSize.height)
+        tableHeighConstraint?.isActive = true
+
+        issuesTableView.tableHeaderView?.backgroundColor = UIColor.white
+    }
+
+    override func viewWillLayoutSubviews() {
```

Comment:
This looks fine, but if you just update the constraints, updateViewConstraints() may be (you'll need to try and check for side effects) a more fitting place to do that.